docker-compose-spark-delta-lake.yml

```
@@ -0,0 +1,50 @@
+version: '3.8'
+
+services:
+  spark-master:
+    image: bde2020/spark-master:3.1.1-hadoop3.2
+    container_name: spark-master
+    ports:
+      - "8080:8080"
+      - "7077:7077"
+    environment:
+      - INIT_DAEMON_STEP=setup_spark
+      - SPARK_MASTER_PORT=7077
+      - SPARK_MASTER_WEBUI_PORT=8080
+      - SPARK_MASTER_LOG=/spark/logs
+    volumes:
+      - ./data:/data
+      - ./logs:/spark/logs
+    networks:
+      - spark-network
+
+  spark-worker:
+    image: bde2020/spark-worker:3.1.1-hadoop3.2
+    container_name: spark-worker
+    environment:
+      - SPARK_WORKER_PORT=8881
+      - SPARK_WORKER_WEBUI_PORT=8081
+      - SPARK_WORKER_LOG=/spark/logs
+      - SPARK_MASTER_URL=spark://spark-master:7077
+    ports:
+      - "8081:8081"
+    depends_on:
+      - spark-master
+    networks:
+      - spark-network
+
+  delta-lake:
+    image: deltaio/delta-docker:0.8.1_2.3.0_arm64
+    container_name: delta-lake
+    ports:
+      - "9000:9000"
+    volumes:
+      - ./data:/data
+      - ./delta-lake:/delta
+    command: ["--s3", "s3a://delta-lake"]
```

```suggestion
    command: ["--s3", "s3://delta-lake"]
```
a는 빼도 될 것 같아요. 이렇게 command를 치면 어떻게 되나요?

---

s3a file system api를 사용하려고 추가하였습니다

---

S3A가 아닌 S3를 사용하면 자체 하둡에서 S3의 저장소를 연결하여 객체를 직접 읽고 쓸 수 없나요? 제가 알기로는 아닌 것 같아서요.
그리고 보내주신 링크 봤는데 저희는 파일이 큰 경우 partitioning을 해서 delta가 알아서 넣을거라 상관없을 것 같아요!

---

넵 제거했습니다~